model/http-common.yaml

```diff
@@ -62,6 +62,25 @@ groups:
           conditionally_required: If and only if one was received/sent.
         brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
         examples: [200]
+      - ref: error.description
+        requirement_level:
+          conditionally_required: If request or response has ended prematurely.
+        examples: ['timeout', 'name_resolution_error', 'server_certificate_invalid']
+        note: |
+          If response status code was sent or received and status indicates an error according to [HTTP span status definition](/docs/http/http-spans.md),
+          `error.description` SHOULD be set to the [Reason Phrase](https://www.rfc-editor.org/rfc/rfc2616.html#section-6.1.1)
```

Comment:
The reason phrase in principle is free-form text. Should this actually include the reason phrase returned by the server, or the RFC standard reason phrase corresponding to the error code?

Reply:
changed to canonical one corresponding to status code